.gitignore

```diff
@@ -3,3 +3,4 @@
 /build
 /.phpcs-cache
 /.phpunit.result.cache
+/.idea
```

Comment:
Device or editor related files/dirs should be ignored on your local machine.
Otherwise this gitignore file might grow very large for all the possible files that specific environments might use.